README.md

```diff
@@ -162,6 +162,30 @@ This means that `*-1.0` and `*-1.1` are configurations for that particular exper
 
 Similarly, `*-3.0` is not reproducible with `*-1.0`, `*-1.1` or `*-2.0`.
 
+### Updating a Configurations Version
+
+To update a configuration that you're using, you can do the following:
+
+```bash
+git fetch --tags  # Make sure your repository has all of them!
+git tag --list release-CONFIG-*  # List all available tags for CONFIG
+```
+
```

Comment:
This doesn't work as a suggestion because of nested code tags ... so:

Find your current version tag:
```
git describe --tags
```
then list all matching tags
```
git tag --list release-CONFIG-*  # List all available tags for CONFIG
```

Reply:
Done in https://github.com/ACCESS-NRI/model-configs-template/pull/5/commits/0da7982a4fd452b5a6e3f935db4d1a8c1eebda40